frontend/src/pages/ApiEntreprise/demarches.json

```
@@ -61,6 +62,7 @@
         "cotisations_msa": true,
         "cotisations_probtp": true,
         "certification_cnetp": true,
+        "certification_opqibi": false,
```

Certification OPQIBI est open data, supprimer cette ligne 
```suggestion
        "certification_opqibi": false,
```